lib/truffler.js

```diff
@@ -141,16 +141,21 @@ Truffler.prototype._run = function(url, options, done) {
 		// Open the URL in PhantomJS
 		openUrl: guardAgainstTimeout(function(next) {
 			options.log.debug('Opening "' + url + '" in PhantomJS');
+
+			state.page.onResourceError = function(resourceError) {
+				this.reason = resourceError.errorString;
```

Comment:
I'm not sure how `this` here is set to the right value, looks a bit magic. Could we maybe just initialise a `var` in the function above it and set that instead? Then we wouldn't need the `bind` on line 158.